src/transformers/modeling_flax_utils.py

```diff
@@ -742,6 +796,22 @@ def from_pretrained(
 
                     # Since we set _raise_exceptions_for_missing_entries=False, we don't get an expection but a None
                     # result when internet is up, the repo and revision exist, but the file does not.
+                    if resolved_archive_file is None and filename == SAFE_WEIGHTS_NAME:
+                        # Maybe the checkpoint is sharded, we try to grab the index name in this case.
+                        resolved_archive_file = cached_file(
+                            pretrained_model_name_or_path, SAFE_WEIGHTS_INDEX_NAME, **cached_file_kwargs
+                        )
+                        if resolved_archive_file is not None:
+                            is_sharded = True
+                            raise NotImplementedError(
```

Comment:
Same here - can we just let the user download the sharded Flax weights in this case? Instead of erroring out when loading the sharded safetensors ones?

Reply:
Correct!

Addressed in https://github.com/huggingface/transformers/pull/27064/commits/1bd29e03c09284653e1d8ae4a8ed426299b99083